quiz.rb

```diff
@@ -2,16 +2,47 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
```

Comment:
Ali, sorry for taking so long to get your score back to you! I got interrupted during my grading this morning, and haven't had a chance to finish until just now.